### PR TITLE
Replace console.log with logger.info

### PR DESCRIPTION
**Pull request by jbcrail**
This PR fixes changes missed during the refactor in #23.

### CODE REVIEW

File: src/plugins/profiling/server/routes/search_flamechart.ts [hunk 1]
```diff
@@ -205,7 +205,7 @@ async function queryFlameGraph(
       stackFrameDocIDs.add(frameID);
     }
   }
-  console.log('unique frames', stackFrameDocIDs.size);
+  logger.info('unique frames' + stackFrameDocIDs.size);
```
Comment (rockdaboot) [inline src/plugins/profiling/server/routes/search_flamechart.ts:208]: Please add a space after '...frames'

File: src/plugins/profiling/server/routes/search_flamechart.ts [hunk 2]
```diff
@@ -164,10 +164,10 @@ async function queryFlameGraph(
     docCount += item.doc_count;
     bucketCount++;
   });
-  console.log('took', resEvents.body.took);
-  console.log('docs', docCount);
-  console.log('total events', totalCount);
-  console.log('unique events', bucketCount);
+  logger.info('query time registered by ES on events ' + resEvents.body.took);
```
Comment (rockdaboot) [inline src/plugins/profiling/server/routes/search_flamechart.ts:167]: Can we add a 'ms' as the 'took unit ?

**Review by rockdaboot — APPROVED** [review]
Otherwise LGTM

**Review by inge4pres — APPROVED** [review]
Same comments as Tim 😄

**Pull request merged**